package.json

```diff
@@ -27,7 +27,7 @@
     "format:go": "gofmt -l -w"
   },
   "devDependencies": {
-    "husky": "^4.3.8",
+    "husky": "^8.0.1",
```

Comment:
No code changes need for 4 major versions?

Reply:
Not from my understanding, nope. it's just a pre-commit hook, and older versions of husky have issues actually running on WSL (Another repo also had this issue for me).

Reply:
There are breaking changes.
https://github.com/typicode/husky-4-to-8
Don't have to use this script. I just googled it. But the way that husky manages the hooks has been changed.
They were stored as json or yaml and now they're stored in `.husky` folder as bash scripts

Reply:
Ah gotcha, will investigate